source/common/quic/envoy_quic_proof_verifier.h

```diff
@@ -39,8 +39,9 @@ using EnvoyQuicProofVerifyContextPtr = std::unique_ptr<EnvoyQuicProofVerifyConte
 // client context config.
 class EnvoyQuicProofVerifier : public EnvoyQuicProofVerifierBase {
 public:
-  explicit EnvoyQuicProofVerifier(Envoy::Ssl::ClientContextSharedPtr&& context)
-      : context_(std::move(context)) {
+  explicit EnvoyQuicProofVerifier(Envoy::Ssl::ClientContextSharedPtr&& context,
```

Comment:
Or alternatively comment here about `accept_untrusted`?

Reply:
yeah I'll point at the docs, which don't mention tests probabably because people do terrible things in prod too =P